### PR TITLE
add support for specifying one or more JSON configs and verbose flag as build step args

### LINKED ISSUE #16: Add support for specifying structure test JSON config location as build step argument

### PR DESCRIPTION
**Pull request by nkubala**
~~partially fixes #16 
support for multiple config files to come in a separate PR.~~

fixes #16.

**UPDATE 10/17**: commit [d3f1da5](https://github.com/GoogleCloudPlatform/runtimes-common/pull/19/commits/d3f1da55addf2c37521f33c2fa9406192013e2a8) adds support for multiple config files.

@dlorenc

### CODE REVIEW

**Comment by nkubala** [comment]
> I'm not too familiar with it, but might getopts be easier?

I tried this at first, but couldn't get it to work the way I wanted it to. under the hood it's doing something almost identical to what I have here, so doesn't really give me any added functionality.

File: structure_tests/structure_test.go [hunk 1]
```diff
@@ -138,27 +140,59 @@ func compileAndRunRegex(regex string, base string, t *testing.T, err string, sho
...
-	flag.StringVar(&configFile, "config", "/workspace/structure_test.json",
-		"path to the .yaml file containing test definitions.")
-	flag.Parse()
+func (a *arrayFlags) Set(value string) error {
+	*a = append(*a, value)
+	return nil
+}
 
+func parseFile(tests *StructureTest, configFile string) {
+	var currentTests StructureTest
```
Comment (dlorenc) [inline structure_tests/structure_test.go]: maybe "testTests" would be more clear?

File: structure_tests/structure_test.go [hunk 2]
```diff
@@ -138,27 +140,59 @@ func compileAndRunRegex(regex string, base string, t *testing.T, err string, sho
...
+
+func combineTests(tests *StructureTest, currentTests *StructureTest) {
+	for _, ct := range currentTests.CommandTests {
+		tests.CommandTests = append(tests.CommandTests, ct)
+	}
+	for _, fet := range currentTests.FileExistenceTests {
+		tests.FileExistenceTests = append(tests.FileExistenceTests, fet)
+	}
+	for _, fct := range currentTests.FileContentTests {
+		tests.FileContentTests = append(tests.FileContentTests, fct)
```
Comment (dlorenc) [inline structure_tests/structure_test.go]: I think you can just do append(tests.FileContentTests, fct...) instead of doing this in a loop.

**Comment by dlorenc** [comment]
Looks like something in the rebase didn't go well?

**Comment by nkubala** [comment]
not the rebase, but I never ran shellcheck locally so didn't realize I was using array in the run script, which don't exist in POSIX sh. converted to construct command string directly from arguments.

File: structure_tests/structure_test.go (hunk 1 above)
Comment (nkubala) [inline structure_tests/structure_test.go]: changed to "tmpTests"

File: structure_tests/types.go [hunk 3]
```diff
@@ -8,6 +8,17 @@ import (
 	"github.com/ghodss/yaml"
 )
 
+type arrayFlags []string
+
+func (a *arrayFlags) String() string {
```
Comment (dlorenc) [inline structure_tests/types.go]: Nit: This should probably return a joined string with all the flags.

**Pull request merged**